config/routes.rb

```diff
@@ -1,11 +1,15 @@
 Rails.application.routes.draw do
+  get 'dashboard/index'
+
+  get 'questions/index'
```

Comment:
We can utilize a namespace here that way we know which routes are API endpoints.